website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -10,7 +10,7 @@ please [migrate][migration-guide], or consult the [legacy docs][legacy-quickstar
 
 ## Prerequisites
 
-- [Install operator-sdk][operator_install] and its prequisites.
+- Go through the [installation guide][install-guide].
 - Access to a Kubernetes v1.11.3+ cluster (v1.16.0+ if using `apiextensions.k8s.io/v1` CRDs).
 - User logged with admin permission. See [how to grant yourself cluster-admin privileges or be logged in as admin][role-based-access-control]
```

Comment:
While you're at it, can you make this change to align tutorials
```suggestion
- User authorized with `cluster-admin` permissions.
```

Reply:
Should we not keep `See [how to grant yourself cluster-admin privileges or be logged in as admin][role-based-access-control]`? I mean, it shows a common mistake. So, it would be : 

- User authorized with `cluster-admin` permissions.See [how to grant yourself cluster-admin privileges or be logged in as admin][role-based-access-control]

Reply:
Thanks for the feedback @camilamacedo86. What do you think @estroz?

Reply:
If I change it, I will make it consistent across all docs.

Reply:
You can continue with my suggestion. I'll make a followup to add back the cluster-admin link as @camilamacedo86 has suggested.

Reply:
ok, updated as per your suggestion @estroz